src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-jvms-page.mdx

```diff
@@ -46,23 +46,22 @@ You can also view all the runtime metrics for a single JVM by clicking the name
     id="jvms-and-metric-types"
     title="Gauges versus counters"
   >
-  We recommend you use the OpenTelemetry Java agent 1.13.0 or higher, which collects JVM memory usage as an async gauge. These agent versions allow you to view metrics in New Relic without the workarounds you'd need to do if you were using older agents.
+We recommend you use the OpenTelemetry Java agent 1.13.0 or higher, which collects JVM memory usage as an async gauge. These agent versions allow you to view metrics in New Relic without the workarounds you'd need to do if you were using older agents.
 
-  If you're using older OpenTelemetry Java agents (versions 1.10.0 thorugh 1.12.0), keep in mind that JVM memory usage switched from being collected as an async gauge to an async up down counter. This has implications on the exported data. Gauges and counters export differently:
+If you're using older OpenTelemetry Java agents (versions 1.10.0 thorugh 1.12.0), keep in mind that JVM memory usage switched from being collected as an async gauge to an async up down counter. This has implications on the exported data. Gauges and counters export differently:
 
-      * Async gauges export as OTLP gauges.
+* Async gauges export as OTLP gauges.
+* Async up down counters export as OTLP non-monotonic sums.
 
-      * Async up down counters export as OTLP non-monotonic sums.
+If you configure your SDK to export your metrics using delta aggregation temporality (which is required for counter and histogram instruments to function with New Relic), that results in async up down counters exported as non-monotonic delta sums. New Relic can't perform any useful analysis of non-monotonic delta sum data.
```

Comment:
```suggestion
If you configure your SDK to export metrics using delta aggregation temporality (which is required for counter and histogram instruments to function with New Relic), that results in async up down counters exported as non-monotonic delta sums. New Relic can't perform any useful analysis of non-monotonic delta sum data.
```

What is "async up down counters"?

Reply:
@a-sassman I had thought these were used as "async up down counters" in the OpenTelemetry spec, but alas, it looks like these are "Asynchronous UpDownCounters."